interface/src/AvatarBookmarks.cpp

```diff
@@ -49,12 +49,12 @@ void addAvatarEntities(const QVariantList& avatarEntities) {
     EntitySimulationPointer entitySimulation = entityTree->getSimulation();
     PhysicalEntitySimulationPointer physicalEntitySimulation = std::static_pointer_cast<PhysicalEntitySimulation>(entitySimulation);
     EntityEditPacketSender* entityPacketSender = physicalEntitySimulation->getPacketSender();
-    QScriptEngine scriptEngine;
+    ScriptEnginePointer scriptEngine = newScriptEngine();
```

Comment:
I'm thinking that both QtScript and V8 engines are fairly heavy-duty objects; in V8 this would likely be creating and destroying an Isolate.  You can explore factoring things out into shared globals as long as you take security (info leakage) and thread-safety into account (V8 will be mutexed to only be active in one thread at a time).  At the current time I'm likely to consider this out of scope for this PR.